llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/TransformOps/LinalgExtTransformOps.cpp

```diff
@@ -142,7 +142,7 @@ LinalgExt::TileToForeachOp::applyToOne(::mlir::linalg::LinalgOp target,
     target->emitError("Failed to tile op to scf.foreach_thread:\n") << target;
     return failure();
   }
-  return SmallVector<Operation *>{result->tileOp, result->tiledOp};
+  return SmallVector<Operation *>{result->tiledOp, result->tileOp};
```

Comment:
please revert this back, other places rely on tiling returning the loop then the tiled op

Reply:
This is inconsistent with the `.td` file. Should I change it in the `.td` file instead?